mypy.ini

```diff
@@ -0,0 +1,9 @@
+[mypy]
```

Comment:
Personally I think that any and all checks we can get are highly valuable here. That is because these tests are so slow that finding bugs late is expensive in terms of development time.